daisi/src/cpps/amr/physical/amr_physical_asset.cpp

```diff
@@ -67,11 +81,43 @@ void AmrPhysicalAsset::sendVehicleStatusUpdateNs3(bool force) {
   socket_->Send(packet);
 }
 
+/// @brief
+/// @return
```

Comment:
Missing comment

The name for the method could be misleading. I assumed that it only gets states but this also modifies ``last_order_states_``.

Reply:
I found out I could do without the workaround.